src/react-demo/Miscellaneous.es6.js

```diff
@@ -1,4 +1,4 @@
-import React, {Component} from 'react'
+import React, { Component } from 'react'
```

Comment:
I'm sorry, prettier updated all of this. 
Lines that you should look at are 351-363